vxquery-core/src/test/java/org/apache/vxquery/datamodel/AbstractPointableTest.java

```diff
@@ -28,7 +28,9 @@
 
     protected void getTaggedValuePointable(Object value, IPointable result) throws IOException {
```

Comment:
Consider the the Comment Node example. The constructor ensures the argument type is correct before calling the builder. The same is true for objects. The constructor will ensure the key is of type string and only pass in the pointable for the string (without the tag). 

Caller - CommentNodeConstructorScalarEvaluator
Builder - CommentNodeBuilder

These comment are trying to get all notions of tags out of the pointable and builder. (No more -1 +1 backed in)